src/plotman/archive.py

```diff
@@ -49,8 +49,14 @@ def compute_priority(phase, gb_free, n_plots):
 
 def get_archdir_freebytes(arch_cfg):
     archdir_freebytes = { }
-    df_cmd = ('ssh %s@%s df -aBK | grep " %s/"' %
-        (arch_cfg['rsyncd_user'], arch_cfg['rsyncd_host'], arch_cfg['rsyncd_path']) )
+    arch_mode = arch_cfg.get('mode', 'remote')
+    if arch_mode == 'remote':
+        df_cmd = ('ssh %s@%s df -BK | grep " %s/"' %
+            (arch_cfg['rsyncd_user'], arch_cfg['rsyncd_host'], arch_cfg['rsyncd_path']) )
+    elif arch_mode == 'local':
+        df_cmd = ('df -BK | grep " %s/"' % arch_cfg['rsyncd_path'] )
+    else:
+        raise KeyError(f'Archive mode must be "remote" or "local" ("{arch_mode}" given). Please inspect config.yaml.')
```

Comment:
This will avoid accidentally invalid quoting by using `repr()`.  Though it also will give you single quotes usually which isn't what you had.  `:|`

https://docs.python.org/3.9/tutorial/inputoutput.html#formatted-string-literals

```suggestion
        raise KeyError(f'Archive mode must be "remote" or "local" ({arch_mode!r} given). Please inspect config.yaml.')
```

---

I personally don't use builtin exceptions and instead make my own.  I often make a base exception per project and inherit all others from that.  But, that's not a mandate here.  Plenty of opinions on that one.